cypress-tags.js

```diff
@@ -12,20 +13,52 @@ const debug = (message, ...rest) =>
     ? console.log(`DEBUG: ${message}`, rest.length ? rest : "")
     : null;
 
-// TODO currently we only work with feature files in cypress/integration folder.
-// It should be easy to base this on the cypress.json configuration - we are happy to take a PR
-// here if you need this functionality!
-const defaultGlob = "cypress/integration/**/*.feature";
+let defaultGlob = "";
+let ignoreGlob = "";
+let usingCypressConf = true;
+
+try {
+  // TODO : curently we don't allow the override of the cypress.json path
+  // maybe we can set this path in the plugin conf (package.json : "cypressConf": "test/cypress.json")
+  const cypressConf = JSON.parse(fs.readFileSync(path.resolve("cypress.json")));
```

Comment:
a bit nitpicky - can we just use require(), since it's json?

Reply:
Done ;) I had to add eslint ignore rules `eslint-disable-next-line import/no-unresolved,global-require` to be able to add the require()